Makefile

```diff
@@ -36,24 +38,24 @@ google/api/%.proto:
 	@test -f $@ || wget --inet4-only -q -O $@ https://raw.githubusercontent.com/googleapis/googleapis/master/$@
 
 .PHONY: build
-## Build runtime Docker image
+## Build runtime container image
 build:
-	docker build -t ${IMG_NAME}:latest --target runtime .
+	${BUILDER} build -t ${IMG_NAME}:latest --target runtime .
```

Comment:
I'm never quite clear when we should use curly braces `${ENGINE}` vs parentheses `$(ENGINE)`. One gets replaced/filled in by Make, the other by Bash?

I see the parens style used more frequently in Makefiles